File: java/src/main/java/DetailFrame.java
public class DetailFrame {

    private String description;
    private String accountSymbol;
    private int frameLength;
    private int frameWidth;
    public void show() {
        System.out.println(description);
    }

    public void setDescription(String description) {
        this.description = description;
    }

    public void setAccountSymbol(String symbol) {
        this.accountSymbol = symbol;
    }

    public String getAccountSymbol() {
        return this.accountSymbol;
    }

    public void setFrameLength(int length) {
        this.frameLength = length;
    }

    public void setFrameWidth(int width) {
        this.frameWidth = width;
    }

    public int getFrameLength() {
        return this.frameLength;
    }

    public int getFrameWidth() {
        return this.frameWidth;
    }
}

File: javascript/src/DetailFrame.js
class DetailFrame {
  constructor({ description, accountSymbol, frameLength, frameWidth }) {
    this._description = description;
    this._accountSymbol = accountSymbol;
    this._frameLength = frameLength;
    this._frameWidth = frameWidth;
  }
  show() {
    console.log(this._description);
  }
  get accountSymbol() {
    return this._accountSymbol;
  }
  get frameWidth() {
    return this._frameWidth;
  }
  get frameLength() {
    return this._frameLength;
  }

  setFrameLength(length) {
    this._frameLength = length;
  }
  setFrameWidth(width) {
    this._frameWidth = width;
  }
  setDescription(description) {
    this._description = description;
  }
  setAccountSymbol(symbol) {
    this._accountSymbol = symbol;
  }
}

module.exports = { DetailFrame };
